Version 2 upgrade, changed to Notification feedback mode. (#10)
eberhardt/moodle-block_coursefeedback: README.md

Since version 1.0.6. there is an option in the plugin settings, to create a global sticky instance.
It will be shown in all course main pages.

Upgrade to v2
==============

All existings blocks are deleted because we want one "system context"-Bblock which we display in all Courses.
We are not allowing single "course-context" blocks because ths could couse doublingissues.

> After the update it is necessary to 

Author: Fehlenden Inhalt ergänzt in dcd16dd73fa1fb1c7ee7826d80fe2a583b42e056

> After the update it is necessary to 

Reviewer: Bitte Rechtschreibung prüfen.

Since version 1.0.6. there is an option in the plugin settings, to create a global sticky instance.
It will be shown in all course main pages.

Upgrade to v2
==============

> All existings blocks are deleted because we want one "system context"-Bblock which we display in all Courses.

Reviewer: Bitte keine pronomiale Sprache in einer Doku. Sowas sollte neutral sein.

Since version 1.0.6. there is an option in the plugin settings, to create a global sticky instance.
It will be shown in all course main pages.

> Upgrade to v2

Reviewer: Mache bitte "Upgrade to Version 2" daraus. Es ist nicht ganz ersichtlich, warum innerhalb des Textes "version" ausschreibst aber dann in einer Überschrift nicht.

Therefore, individual "Course Context" blocks are no longer allowed as they could conflict with the system block.
Also for this reason only managers are allowed to add or delete the block. There can only be exactly one block in each course.

To make things as easy as possible, the necessary system block will be added automatically when upgrading to version 2 or installing the block.
This means that the block only needs to be added manually if it has been deleted beforehand.

Here is how you would add the block manually:   
  1.  First, start on the frontpage of your Moodle site ("Home"). Make sure that editing is turned on, and click "+ Add a block".
  2.  Once the block has been added to the page, click the settings icon, and click "Configure". 
  3.  On the block settings page, under "Where this block appears", select "Display throughout the entire site".
  4.  Next, open up one of the existing courses on your Moodle site (it doesn't matter which course).
  5.  Now, click the block settings icon on the block again, and click "Configure".
  6.  Under "Where this block appears", change the setting to "Any type of course main page"

> (instruction source: https://createdbycocoon.com/knowledge/adding-block-all-courses-moodle)

Reviewer: Ein paar Änderungswünsche.

Reviewer: ```suggestion
If the block is accidentally deleted, it can be added by following steps[^1]:
  1. start on the frontpage of your Moodle site ("Home"). Make sure that editing is enabled and click "+ Add a block".
  1. once the block has been added to the page, click on the gear icon and then click on "Configure".
  1. in the settings, under "Where this block appears", select "Display throughout the entire site".
  1. now open one of the existing courses on your Moodle site (it doesn't matter which course it is).
  1. click on the block settings icon for the block again and then click on "Configure".
  1. under "Where this block appears" change the setting to "Any type of course main page"

[^1]: Source: https://createdbycocoon.com/knowledge/adding-block-all-courses-moodle
```

Since version 1.0.6. there is an option in the plugin settings, to create a global sticky instance.
It will be shown in all course main pages.

> Upgrade to version 2

Reviewer: "Version"

Reviewer: ```suggestion
Upgrade to Version 2
```

Reviewer: Die ganze Readme ist halt auch schon Jahre alt. Ich würde da jetzt auch einiges anders schreiben, als ich es damals getan habe. Da muss ich dann mal über das Alte rüber gehen.

==============
For version 2, all existing blocks are deleted because one "system context"-block is wanted which will be displayed in all Courses.
Single "course-context" blocks are not allowed because this could cause duplicate issues. 
Only mangers are allowed now to add or delete the block. Exactly one block in each course is wanted.
To make things as easy as possible the blcoks are automatically added while upgradind to version 2 or installing of the block.

> Since the block is automatically added, manually adding the block is only needed if the block was manually deleted before for some reason.

Reviewer: Leider sind immer noch Rechtschreibfehler bzw. Typos drin.

> Since the block is automatically added, manually adding the block is only needed if the block was manually deleted before for some reason.

Reviewer: Leider sind immernoch Rechtschreibfehler drin.

> Since the block is automatically added, manually adding the block is only needed if the block was manually deleted before for some reason.

Reviewer: Leider sind immernoch Rechtschreibfehler drin.

Reviewer: For version 2, all existing blocks will be deleted as the rendering is now achieved via a "system context" block that is visible in all courses.
Therefore, individual "Course Context" blocks are no longer allowed as they could conflict with the system block.
Also for this reason only managers are allowed to add or delete the block. There can only be exactly one block in each course.

To make things as easy as possible, the necessary system block will be added automatically when upgrading to version 2 or installing the block.
This means that the block only needs to be added manually if it has been deleted beforehand.

Reviewer: ```suggestion
For version 2, all existing blocks will be deleted as the rendering is now achieved via a "system context" block that is visible in all courses.
Therefore, individual "Course Context" blocks are no longer allowed as they could conflict with the system block.
Also for this reason only managers are allowed to add or delete the block. There can only be exactly one block in each course.

To make things as easy as possible, the necessary system block will be added automatically when upgrading to version 2 or installing the block.
This means that the block only needs to be added manually if it has been deleted beforehand.
```
